pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -201,28 +192,3 @@ func isSelfNodeClientCert(csr *capi.CertificateSigningRequest, x509cr *x509.Cert
 	}
 	return true
 }
-
```

Comment:
Unrelated to this change but what do you think about adding debug logging in the recognizer's about the exact condition which led to CSR not being approved?

Reply:
Events are better for exposing reasons to end users. Would this be for certs that are recognized but the subject access review is denied?

Reply:
Debug logging might be ok but not all CSRs are expected to be approved by this controller. Unrecognized CSRs are expected under normal circumstances.